src/json-path.ts

```diff
@@ -71,9 +71,7 @@ export interface WFF {
   rhs?: Group | number | "last"
 }
 
-export interface Group {
-  wff: WFF
-}
+export interface Group extends WFF {}
```

Comment:
This can be simplified as `export type Group = WFF`

Reply:
OK, GTK